dlpy/network.py

```diff
@@ -726,22 +745,68 @@ def load_weights_from_file(self, path, format_type='KERAS'):
             contains the weight table.
         format_type : KERAS, CAFFE
             Specifies the source framework for the weights file
+        data_spec: list of :class:`DataSpec`, optional
+            data specification for input and output layer(s)            
 
         '''
         cas_lib_name, file_name = caslibify(self.conn, path, task='load')
 
-        self._retrieve_('deeplearn.dlimportmodelweights', model=self.model_table,
-                        modelWeights=dict(replace=True,
-                                          name=self.model_name + '_weights'),
-                        formatType=format_type, weightFilePath=file_name,
-                        caslib=cas_lib_name)
+        if (data_spec):
```

Comment:
BTW, Python doesn't need parens around 'if' conditions.  I see that in various places in this code.

Reply:
Agreed, but I find it helps readability (at least for me).  If this is a style problem, I can certainly make the change, but otherwise would prefer to leave it as it is.